Description: specifying vendoring and updating classpath in the jar manifest
Author: Sylvestre Ledru <sylvestre@debian.org>
Forwarded: not-needed
Last-Update: 2023-04-17

--- a/make/joglversion
+++ b/make/joglversion
@@ -2,7 +2,7 @@
 Application-Name: Java Bindings for OpenGL
 Specification-Title: Java Bindings for OpenGL API Specification
 Specification-Version: @BASEVERSION@
-Specification-Vendor: JogAmp Community
+Specification-Vendor: JogAmp Community - Debian package
 Implementation-Title: Java Bindings for OpenGL Runtime Environment
 Implementation-Version: @VERSION@
 Implementation-Build: @BUILD_VERSION@
@@ -15,6 +15,7 @@
 Trusted-Library: true
 Permissions: all-permissions
 Application-Library-Allowable-Codebase: *
+Class-Path: /usr/share/java/gluegen2-rt.jar
 Add-Opens: java.desktop/sun.awt java.desktop/sun.java2d
 @JAR_CODEBASE_TAG@
 
